readme.md

```diff
@@ -30,8 +30,8 @@ Make all the changes smooth on hover (during 300ms):
 ## Checklist
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://Pavlo-lab-max-Kopalko.github.io/layout_catalog/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="554" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/5b107c0c-2f98-45b2-a0f6-75d4e4f509ff">

Reply:
Fancy that. As usual my code has a lot of mistakes. Bbu no one now. It's ipmosible.

Reply:
By the way, mate-platform show me some tests have failed.